<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_book_page</class>
 <widget class="QWidget" name="add_book_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>576</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="add_book_label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>图书导入</string>
   </property>
  </widget>
  <widget class="QLabel" name="single_import">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>390</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>单册导入</string>
   </property>
  </widget>
  <widget class="QLabel" name="batch_import">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>批量导入</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="path">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>305</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>请输入目标文件路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bi_confirm">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>350</y>
     <width>95</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>95</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>95</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>411</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QLineEdit" name="bno">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>书号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="category">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>类别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="title">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>书名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="press">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>出版社</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QLineEdit" name="year">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>年份</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="author">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>作者</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="price">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>价格</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="total">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>数量</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="return_button">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>500</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="si_confirm">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>500</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>95</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="success_sign">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>导入成功！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
